src/shared/utils/analytics.ts

```diff
@@ -43,13 +43,24 @@ export function initSentry() {
     integrations: [new BrowserTracing()],
     tracesSampleRate: 1.0,
     environment: WALLET_ENVIRONMENT,
+    beforeSendTransaction(event) {
+      // Not currently using these transaction types, navigation is covered by
+      // segment, and they heavily contribute towards the quota.
+      if (event.contexts?.trace?.op === 'pageload' || event.contexts?.trace?.op === 'navigation') {
+        return null;
+      }
+
+      return event;
+    },
```

Comment:
Isn't this controlled by this argument? https://docs.sentry.io/platforms/javascript/configuration/options/#auto-session-tracking

Reply:
Good one